emissions.md

```diff
@@ -90,10 +117,10 @@ where:
 
 | | |
 | :------------- |:-------------|
-| F | Emissions factor: base-year emissions per unit activity |
-| A | Activity level (e.g., output of a technology) |
-| MAC | Marginal Abatement Cost Curve |
-| Eprice | Emissions Price |
+| `F` | Emissions factor: base-year emissions per unit activity |
+| `A` | Activity level (e.g., output of a technology) |
+| `MAC` | Marginal Abatement Cost Curve |
+| `Eprice` | Emissions Price |
 
 Non-CO<sub>2</sub> GHG emissions are proportional to the activity except for any reductions in emission intensity due to the MAC curve. As noted above, the MAC curves are assigned to a wide variety of technologies, mapped directly from [EPA 2013](#epa2013). Under a carbon policy, emissions are reduced by an amount determined by the MAC curve.
```

Comment:
Change from EPA 2013 to EPA 2019 (and update the reference) [https://www.epa.gov/global-mitigation-non-co2-greenhouse-gases/global-non-co2-greenhouse-gas-emission-projections](url)